docs/Configuration.md

```diff
@@ -126,30 +130,30 @@ specVersion: "1.0"
 kind: extension
 type: project-shim
 metadata:
-  name: <name of project shim extension>
+	name: <name of project shim extension>
 shims:
-  configurations:
-    <module name (id)>:
-      specVersion: "1.0",
-      type: <project type>
-      metadata:
-        name: <project name>
-    <module name (id)>:
-      specVersion: "1.0",
-      type: <project type>
-      metadata:
-        name: <project name>
-  dependencies:
-    <module name (id)>:
-      - <module name (id)>
-      - <module name (id)>
-      - <module name (id)>
-  collections:
-    <module name>:
-      modules:
-        <id>: <relative path>
-        <id>: <relative path>
-        <id>: <relative path>
+	configurations:
```

Comment:
isn't the convention for yaml files "two spaces"?

Reply:
My IDE mixed things up in this file... 😞